src/Symfony/Bundle/FrameworkBundle/Tests/Functional/app/AutowiringTypes/bundles.php

```diff
@@ -11,8 +11,10 @@
 
 use Symfony\Bundle\FrameworkBundle\Tests\Functional\Bundle\TestBundle\TestBundle;
 use Symfony\Bundle\FrameworkBundle\FrameworkBundle;
+use Symfony\Bundle\SecurityBundle\SecurityBundle;
 
 return array(
     new FrameworkBundle(),
+    new SecurityBundle(),
```

Comment:
I missed that, but indeed, that's not good. @chalasr Can you move it to SecurityBundle?